internal/emscripten/emscripten.go

```diff
@@ -89,8 +109,17 @@ func (v *InvokeFunc) Call(ctx context.Context, mod api.Module, stack []uint64) {
 		panic(err)
 	}
 
-	err = f.CallWithStack(ctx, stack)
+	stackSave, err := mod.ExportedFunction("stackSave").Call(ctx)
```

Comment:
nit: I'd call this `stackPointer`, or `savedStack`; overloading the function name threw me off a bit.

Reply:
Changed :)